CoreHistogramView/src/au/gov/asd/tac/constellation/views/histogram/BinComparator.java

```diff
@@ -99,9 +99,9 @@ public BinComparator getReverse() {
     TOTAL_COUNT("Total Count", true, false) {
         @Override
         public int compare(Bin o1, Bin o2) {
```

Comment:
While making changes to this class, can you also add finals to the method declarations.